widgets/Hubspot-batchsink.json

```diff
@@ -4,7 +4,38 @@
   },
   "configuration-groups": [
     {
-      "label": "Basic",
+      "label": "Authorization",
+      "properties": [
+        {
+          "name": "authorizationMethod",
+          "label": "Authorization method",
+          "widget-type": "select",
+          "widget-attributes": {
+            "values": [
+              "API key",
```

Comment:
nit: capitalize each word of property title, i.e. `API Key`, `Private App Access Token`, same for all below

Reply:
Fixed